doc/devel/testing.rst

```diff
@@ -20,6 +21,13 @@ infrastructure are in :mod:`matplotlib.testing`.
 Requirements
 ------------
 
+We strongly recommend developing new features in a seperate virtualenv.
+
+Install all the matplotlib dependencies_, and install matplotlib into the
+virtualenv with ``pip install -e matplotlib`` (but note the changes needed to
```

Comment:
OK, this isn't really my workflow.  If you are developing, you need to pull down master from GitHub and `pip install -e .` from the GitHub directory.  I also think that @anntzer just had a PR like this, or at least one that explicitly said how to `pip install`.

Reply:
See #9291 and use of `python -mpip install -e .`